dpnp/tests/test_umath.py

```diff
@@ -667,32 +600,33 @@ def test_out(self, func_params, dtype):
         expected = getattr(numpy, func)(a)
 
         ia = dpnp.array(a)
-        out_dtype = _get_output_data_type(dtype)
-        iout = dpnp.empty(expected.shape, dtype=out_dtype)
+        if func == "square":
+            dt_out = numpy.int8 if dtype == dpnp.bool else dtype
```

Comment:
Can it be moved inside `_get_output_data_type`?
In general, probably we need to improve implementation of `_get_output_data_type`  and to parse output of `getattr(numpy, func).types` to obtain the expected `out` dtype.